include/common/debug.h

```diff
@@ -47,5 +49,8 @@ noreturn void panic(const char *fmt, ...) __printf(1, 2);
 #else
 #define debug(...) ((void)0)
 #endif
-
+#if TEST
+#define test(...)  log(LOG_STRING_DEBUG __VA_ARGS__)
+#endif
 #endif /* DEBUG_H */
+
```

Comment:
The blank line goes _above_ the `#endif`

Reply:
done